docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc

```diff
@@ -69,12 +75,14 @@ If you see the serial number for your instance and want to change it to a descri
 [#id3c964e17-24c6-4e7c-9a47-adae096cc88d]
 === Log in Using the Prisma Cloud URL
 
-If you have accepted the EULA and have a Palo Alto Networks CSP account, you can access the Prisma Cloud app directly.
+Follow the steps below to access the Prisma Cloud app directly.
 
 [.procedure]
 . Access the Prisma Cloud App.
 +
-Enter your username and password. The URL is *<Prisma Cloud URL>*/auth/signin*
+Navigate to https://appX.prismacloud.io/login (where X is the instance where your tenant is provisioned, for example app3). Use the xref:#domain-list[domain-list] list above for domain specific URLs.
+
+. Enter your username and password. 
 
 . *Sign in*.
 +
```

Comment:
The login page now offers multiple options here which I think we should disclose. In addition to the "Sign In" button, we have a "Sign in with SSO" button which is not discussed here. And we should probably cover the two distinct login flows behind the SSO button as well (OIDC, PANW-SSO).

Reply:
I updated with all the login options we're offering now. Please take a look @ftbrecordspan.